build-source-tarball.sh

```diff
@@ -3,7 +3,7 @@ set -euo pipefail
 IFS=$'\n\t'
 
 usage() {
-    echo "usage: $0 <path-to-tarball-root> [--skip-build] [--enable-leak-detection] [--skip-prebuilt-check] [-- [extra build.sh args]]"
+    echo "usage: $0 <path-to-tarball-root> [--skip-build] [--enable-leak-detection] [--skip-prebuilt-check] [--custom-ref-packages <dir>] [--custom-previously-built-packages <dir>] [-- [extra build.sh args]]"
```

Comment:
This is pretty much bikeshedding, so apologies in advance. I wonder if we can make the options a bit easier to use by renaming them to `with`:

```
     ./build-source-tarball.sh <path-to-tarball-root> \
         --with-sdk /path/to/system/dotnet \
         --with-ref-packages /path/to/source-build-reference-packages \
         --with-packages /path/to/source-build/packages
```

The [`--with`-style options](https://autotools.io/autoconf/arguments.html) are commonly used by autotools and are fairly standard and obvious to developers on *nix. What do you think?

Reply:
No problem, that seems reasonable to me.